### DISCUSSION TITLE
Using strapi on windows - setup errors

### DISCUSSION THREAD

**Discussion by netanelmaroof** [strapi/strapi#7707] [Legacy Topics]
I loved strapi when i was developing on a mac, but now that I'm using windows for some reason I can't install Strapi's dependencies; I get errors no matter what settings I choose. Anyone else having this issue? Any ideas on how to fix this?

**Comment by MattieBelt** [comment]
Hey @netanelmaroof great to hear you loved working with Strapi, I want to help you out, but I'll need some more context for that. Can you please share the errors log and your system specs? Also could you try out if a fresh project also fails to run correctly?

- Node.js version: \
- NPM version:
- Strapi version: \
- Database (version):
- Operating system: Windows \<version>
